docs/ops/activation/Swish_4.md

```diff
@@ -0,0 +1,53 @@
+## Swish <a name="Swish"></a>
+
+**Versioned name**: *Swish-4*
+
+**Category**: *Activation*
+
+**Short description**: Swish takes one input tensor and produces output tensor where the Swish function is applied to the tensor elementwise.
+
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: 
+
+    \f[
+    Swish( x ) = x*sigmoid{beta*x} where sigmoid{x} = 1.0/{1.0+e^{-x}}.
```

Comment:
Have you checked that the formula is correctly rendered? I think that it will be broken. 

Let's make the formula look like this `Swish(x) = x / (1.0 + e^{-beta * x})` without mentioning the sigmoid.

Reply:
Done.